lib/bitvec.h

```diff
@@ -45,13 +45,10 @@ limitations under the License.
 #if defined(__GNUC__) || defined(__clang__)
 /* use builtin count leading/trailing bits of type-approprite size */
 static inline int builtin_ctz(unsigned x) { return __builtin_ctz(x); }
-static inline int builtin_ctz(unsigned long x) { return __builtin_ctzl(x); }
 static inline int builtin_ctz(unsigned long long x) { return __builtin_ctzll(x); }
 static inline int builtin_clz(unsigned x) { return __builtin_clz(x); }
-static inline int builtin_clz(unsigned long x) { return __builtin_clzl(x); }
```

Comment:
If we ever run into a machine where sizeof(long) != sizeof(int) and sizeof(long) != sizeof(long long), we'll need this declaration for correctness...